shuup/simple_cms/models.py

```diff
@@ -54,11 +66,11 @@ def visible(self, shop, dt=None, user=None):
         )
         if user and not user.is_anonymous():
             q |= Q(created_by=user)
-        qs = self.for_shop(shop).filter(q)
+        qs = self.filter(q).for_shop(shop)
         return qs
 
     def for_shop(self, shop):
-        return self.filter(shop=shop)
+        return self.get_queryset().for_shop(shop=shop)
```

Comment:
Doesn't make sense to add this `get_queryset()` just to replace `filter(deleted=False)`?

Reply:
reverted `SupplierQueryset ` and added `not_deleted()` method